content/booklet.md

```diff
@@ -0,0 +1,178 @@
+Even NWERC 2024 is organised by members of the study association W.I.S.V. 'Christiaan Huygens'!
+
+![Delft City Center Map](/delft-city-center-map.jpg)
+*Source: [OnTheWorldMap](https://ontheworldmap.com/netherlands/city/delft/delft-city-center-map.html)*
+
+![Northpool](/northpool_ad.png)
+
+## Systems
+During the contest, the computer is your most valuable resource (perhaps apart from coffee).
+A brief explanation of our systems can be found on this page.
+
+### Hardware
+Each team will have exactly one
+workstation.
+The only operating system available to you will be **Ubuntu 24.04** with **Xfce**.
+
+### Software
+
+The jury will run your submissions on the same system with the same compilers.
+During the contest, you will be able to choose between the languages **C, C++, Python, Java and Kotlin**.
+Using different languages for different problems is allowed.
+For a complete list of available IDEs, editors, language versions and compiler commands, see [2024.nwerc.eu/systems](/systems).
+
+### During the contest
+An introductory presentation on the systems will be given on Saturday; please refer to the schedule for the exact time.
+At this presentation, do not hesitate to ask any questions.
+Furthermore, if you stumble upon any problems with the system during the test session or contest, let us know and we will be more than happy to assist you.
+
+![Da Vinci](/da_vinci_ad.png)
+
+## Jury and Rules
+We follow the World Finals rules, so a team is not allowed to bring any reference materials such as books, program listings, or notes (exceptfor the TRDs)!
+You may not bring any electronic device or machinereadable media.
+You may bring pens or pencils, but not paper.
+Winter coats, bags etc. should not be brought to the contest area.
+
+Each contestant may bring a copy of a Team Reference Document (TRD).
+This may contain up to 25 pages of reference materials, singlesided, letter or A4 size.
+The pages must be numbered in the upper right-hand corner and contain the university name in the upper lefthand corner.
+Text and illustrations must be readable by a person with correctable eyesight without magnification from a distance 0.5 meter.
+It may include handwritten comments and corrections on the front of the pages only.
+The document should be in a notebook or folder with the name of your institution on front.
+
+Each team member may bring one printed, unannotated natural language dictionary.
+You may bring mascots such as stuffed animals or party hats provided they do not violate the above constraints.
+If there is any other item you need, please contact the Head of Jury.
+
+Teams are allowed to bring one keyboard.
+Note that keyboards with pre-programmed macro keys are not allowed.
+Wireless keyboards are also not allowed.
+One mouse is allowed too, but is subject to the same macro rule as with the keyboard.
+For more jury advice, see: [2024.nwerc.eu/jury-advice](/jury-advice).
+
+### Team Composition
```

Comment:
Should this be in here?

Reply:
Also related to your suggestion to add the "Conduct during contest": I'm not 100% sure if it's still useful to repeat the rules in the digital booklet, when we can simply link to `/rules` :stuck_out_tongue:

Reply:
The digital booklet becomes a TL:DR of the whole website. For someone not participating this might be useful information, so we could leave it here.